### PR TITLE
adding parser and tests for cedars sinai

### PR DESCRIPTION
**Pull request by princessbeane**
-Added parser for cedars-sinai
-Added test for cedars-sinai parser

### CODE REVIEW

File: src/chargemaster_parsers/parsers/cedars_sinai.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    def parse_artifacts(self, artifacts):
+        for artifact_url, artifact in artifacts.items():
+            wb = openpyxl.load_workbook(artifact)
+            charge_code_column = None
+            charge_code_description_column = None
+            cpt_hcpcs_code_column = None
+            op_charge_column = None
+            ip_charge_column = None
+
+            for _, row in enumerate(wb.worksheets[0].iter_rows(min_row=5)):
```
Comment (jacobschaer) [inline src/chargemaster_parsers/parsers/cedars_sinai.py:26]: You don't need to enumerate if you're going to throw away the index parameter.
```suggestion
            for row in wb.worksheets[0].iter_rows(min_row=5):
```
The min_row flag is pretty cool though, I didn't see that when I glanced at the docs.

File: tests/test_cedars_sinai.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            location = 'all',
+            procedure_description = "HB IV INFUS HYDRATION 31-60 MIN",
+            gross_charge = float(2161.71),
+            in_patient = False
+        ),
+        ChargeMasterEntry(
+            procedure_identifier = 2600001,
+            location = 'all',
+            procedure_description = "HB IV INFUS HYDRATION 31-60 MIN",
+            gross_charge = float(2811.66),
```
Comment (jacobschaer) [inline tests/test_cedars_sinai.py:55]: Python will assume a float type from a decimal literal, but it doesn't hurt. See Line 64 it works the same.

**Review by jacobschaer — COMMENTED** [review]
Looks good. Just two nit's. Will merge later or if you want to fix anything.

File: tests/test_cedars_sinai.py (hunk 2 above)
Comment (princessbeane) [reply tests/test_cedars_sinai.py]: fixed

File: src/chargemaster_parsers/parsers/cedars_sinai.py (hunk 1 above)
Comment (princessbeane) [reply src/chargemaster_parsers/parsers/cedars_sinai.py]: fixed

**Pull request merged**